CMakeLists.txt

```diff
@@ -98,6 +98,9 @@ if ("${CMAKE_HOST_SYSTEM_PROCESSOR}" STREQUAL "x86_64")
     add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/mcsema/Arch/X86/Runtime)
 endif()
 
+#TODO(artem): this may need some logic to select only ABIs compatible with current os/arch
+add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/OS/Linux)
```

Comment:
Lets move his to `mcsema/OS/Linux`, putting it top-level was a mistake on my end.